app/Managers/PostManager.php

```diff
@@ -61,7 +61,7 @@ public function create($data, $signupId, $authenticatedUserRole = null)
         );
 
         // Save the new post in Customer.io, via Blink.
-        if (config('features.blink') && $shouldSendToCustomerIo) {
+        if ($shouldSendToCustomerIo) {
```

Comment:
Since we [check the feature flag within the `CustomerIo` class](https://github.com/DoSomething/rogue/blob/1f7f1f6a9113d078f9304a3e837653bb3eb73795/app/Services/CustomerIo.php#L47-L55), this check is unnecessary. By removing it, we also get these logged on local environments.